src/adapters/analytics/roxot.js

```diff
@@ -1,19 +1,94 @@
-import { ajax } from 'src/ajax';
-import adapter from 'AnalyticsAdapter';
+import {ajax} from "src/ajax";
+import adapter from "AnalyticsAdapter";
+import CONSTANTS from "src/constants.json";
 
 const utils = require('../../utils');
 
-const url = '//d.rxthdr.com/analytics';
+const url = '//pa.rxthdr.com/analytic';
 const analyticsType = 'endpoint';
 
-export default Object.assign(adapter(
-  {
-    url,
-    analyticsType
+let auctionInitConst = CONSTANTS.EVENTS.AUCTION_INIT;
+let auctionEndConst = CONSTANTS.EVENTS.AUCTION_END;
+let bidWonConst = CONSTANTS.EVENTS.BID_WON;
+
+let initOptions = {publisherIds: []};
+let bidWon = {options: {}, events: []};
+let eventStack = {options: {}, events: []};
+
+let auctionStatus = 'not_started';
+
+function checkOptions() {
+  if (typeof initOptions.publisherIds === 'undefined') {
+    return false;
   }
-  ),
+
+  return initOptions.publisherIds.length > 0;
+}
+
+function buildBidWon(eventType, args) {
+  bidWon.options = initOptions;
+  bidWon.events = {args: args, eventType: eventType};
+}
+
+function buildEventStack() {
+  eventStack.options = initOptions;
+}
+
+function send(eventType, data, sendDataType) {
+  let fullUrl = url + '?publisherIds[]=' + initOptions.publisherIds.join('&publisherIds[]=') + '&host=' + window.location.hostname;
+
+  ajax(
+    fullUrl,
+    (result) => utils.logInfo('Event ' + eventType + ' sent ' + sendDataType + ' to roxot prebid analityc with result' + result),
+    JSON.stringify(data)
+  );
+}
+
+function pushEvent(eventType, args) {
+  eventStack.events.push({eventType, args});
+}
+
+function flushEvents() {
+  eventStack.events = [];
+}
+
+let roxotAdapter = utils.extend(adapter({url, analyticsType}),
   {
-    track({ eventType, args }) {
-      ajax(url, (result) => utils.logInfo('Event ' + eventType + ' sent to roxot analytics with result ' + result), JSON.stringify({ eventType, args }));
+    track({eventType, args}) {
+      if (!checkOptions()) {
+        return;
+      }
+
+      args.ad = "";
```

Comment:
Probably best to check for the existence of `args` first, if `args` is undefined this causes a `TypeError` originating in event.js to be thrown